Assets/Scripts/Game/MagicAndEffects/Effects/Illusion/InvisibilityNormal.cs

```diff
@@ -19,25 +19,41 @@ namespace DaggerfallWorkshop.Game.MagicAndEffects.MagicEffects
     /// </summary>
     public class InvisibilityNormal : ConcealmentEffect
     {
+        public static readonly string PublicKey = "Invisibility-Normal";
+
         public override void SetProperties()
         {
-            properties.Key = "Invisibility-Normal";
```

Comment:
Here's an example of making the EntityEffect key publicly available to support multi-effect potions while still keeping all potion recipes in EntityEffect classes as you wanted. We should discuss if there's a better way of doing this.